cpp/src/arrow/array/array_nested.h

```diff
@@ -125,13 +125,19 @@ class ARROW_EXPORT ListArray : public BaseListArray<ListType> {
   /// \param[in] values Array containing list values
   /// \param[in] pool MemoryPool in case new offsets array needs to be
```

Comment:
Please mention above that either a offsets Array's null bitmap can be present or an explicit null_bitmap, but not both.